sktime/transformations/tests/test_fitintransform.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Unit tests of Imputer functionality."""
+
+__author__ = ["aiwalter"]
+__all__ = []
+
+from pandas.testing import assert_frame_equal
+
+from sktime.forecasting.model_selection import temporal_train_test_split
+from sktime.transformations.compose import FitInTransform
+from sktime.transformations.series.exponent import ExponentTransformer
+from sktime.utils._testing.forecasting import make_forecasting_problem
+
+y, X = make_forecasting_problem(make_X=True)
+y_train, y_test, X_train, X_test = temporal_train_test_split(y, X)
+
+
+def test_transform_fitintransform():
+    """Test fit/transform against ExponentTransformer."""
+    fitintransform = FitInTransform(ExponentTransformer())
```

Comment:
same here, you want to use the boxcox or similar transform that does not have empty `_fit`.
why not use `create_test_instance`?